### ISSUE TITLE
scylla_setup - "Do you want to enable ScyllaDB services" is asked twice

### ISSUE THREAD

**Issue by slivne** [scylladb/scylla#1334]
*Installation details*
Scylla version (or git commit hash): 1.2.rc2

```
ubuntu@ip-10-45-9-102:~$ sudo /usr/lib/scylla/scylla_setup 
Do you want to run kernel version check?
yes/no: no
Do you want to enable ScyllaDB services?
yes/no: yes
Do you verify ScyllaDB packages installed?
yes/no: no
Do you want to enable ScyllaDB services?
yes/no: yes
Do you want to disable SELinux?
```

**Comment by avikivity** [comment]
btw, "enable" is vague.

better wording is "do you want ScyllaDB to start up automatically as a service?"
and "start ScyllaDB now?"

**Comment by slivne** [comment]
fixed by 0cf81444856a76df06cfc388b496e276f878ff05
